habitat-lab/habitat/datasets/rearrange/samplers/receptacle.py

```diff
@@ -984,7 +989,8 @@ def get_navigable_receptacles(
                     sim=sim,
                     point=point,
                     height=max_access_height,
-                    nav_to_min_distance=nav_to_min_distance,
+                    # nav_to_min_distance=nav_to_min_distance,
+                    nav_to_min_distance=-1,
```

Comment:
By the way, we'd better revert this change.

Reply:
I currently need this part for generation, and I will fix it in the future. I will set it as a TODO.